tools/create_windows_sg/terraform.tfstate

```diff
@@ -1,7 +1,7 @@
 {
     "version": 3,
     "terraform_version": "0.7.4",
-    "serial": 0,
```

Comment:
Storing `.tfstate` in source control is frowned upon. I think this is probably here b/c it's easiest to commit to source while getting started, and there are no _real_ secrets in there.  In the future TF has support for storing state in S3 (https://www.terraform.io/docs/state/remote/).  I don't think the same thing works w/ Azure Storage. hmm.